docs/management/numeral.asciidoc

```diff
@@ -0,0 +1,183 @@
+[[numeral]]
+== Numeral Formatting
+
+Numeral formatting in {kib} is done through a pattern-based syntax.
+These patterns express common number formats in a concise way, similar
+to date formatting. While these patterns are originally based on Numeral.js,
+they are now maintained by {kib}.
+
+Numeral formatting patterns are used in multiple places in {kib}, including:
+
+* <<advanced-options, Advanced settings>>
+* <<field-formatters-numeric, Index pattern formatters>>
+* <<TSVB, TSVB>>
+* <<canvas, Canvas>>
+
+The simplest pattern format is `0`, and the default {kib} pattern is `0,0.[000]`.
+The numeral pattern syntax expresses:
+
+Number of decimal places:: The `.` character turns on the option to show decimal
+places using a locale-specific decimal separator, most often `.` or `,`.
+To add trailing zeroes such as `5.00`, use a pattern like `0.00`.
+To have optional zeroes, use the `[]` characters. Examples below.
+Thousands separator:: The thousands separator `,` turns on the option to group
+thousands using a locale-specific separator. The separator is most often `,` or `.`,
+and sometimes ` `.
+Accounting notation:: Putting parentheses around your format like `(0.00)` will use accounting notation to show negative numbers.
+
+The display of these patterns is affected by the <<kibana-general-settings, advanced setting>> `format:number:defaultLocale`.
+The default locale is `en`, but some examples will specify that they are using an alternate locale.
+
+Most basic examples:
+
+|===
+| **Input** | **Pattern** | **Locale** | **Output**
+| 10000.23 | 0,0 | en (English) | 10,000
+| 10000.23 | 0.0 | en (English) | 10000.0
+| 10000.23 | 0,0.0 | fr (French) | 10 000,0
```

Comment:
The output here looks wrong, should be `10000.2`/`10 000,2` right?

Reply:
@spalger Thanks for catching!